components/movementsensor/gpsrtk/gpsrtk.go

```diff
@@ -133,7 +133,7 @@ func init() {
 			cfg config.Component,
 			logger golog.Logger,
 		) (interface{}, error) {
-			return newRTKStation(ctx, deps, cfg, logger)
+			return newRTKMovementSensor(ctx, deps, cfg, logger)
```

Comment:
I've added a guard, not totally convinced the pattern in `base_test.go` helps here, were' missing the concept of a `movementsensor.Local`. There is a way to rewrite it so that the creation function is based on a switch case depending on the resource name, but I'm not sure that's a good way to do it.

Reply:
Ugh you're right and I'm sorry I led you astray there. The test should be more like https://github.com/viamrobotics/rdk/blob/main/robot/session_test.go#L156 where we create a config, refer to the model, construct the robot, and get the component out of it. That's fairly end to end. You'll want to move this test into a new file with package `gpsrtk_test` so that you can import robot. Sorry!